include/mbedtls/ssl.h

```diff
@@ -1359,7 +1359,7 @@ struct mbedtls_ssl_config
 #endif /* MBEDTLS_SSL_ASYNC_PRIVATE */
 
 #if defined(MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED)
-    const int *MBEDTLS_PRIVATE(sig_hashes);          /*!< allowed signature hashes           */
+    const uint16_t *MBEDTLS_PRIVATE(sig_algs);          /*!< allowed signature hashes           */
```

Comment:
variable is sig_algs and the comment is signature hashes.. Should we allign them?